nautobot/ipam/models.py

```diff
@@ -613,21 +613,26 @@ def get_first_available_ip(self):
         return "{}/{}".format(next(available_ips.__iter__()), self.prefix.prefixlen)
 
     def get_utilization(self):
-        """
-        Determine the utilization of the prefix and return it as a percentage. For Prefixes with a status of
-        "container", calculate utilization based on child prefixes. For all others, count child IP addresses.
+        """Get the child prefix size and parent size.
+
+        Get the child prefix size and parent prefix size return them as a tuple. For Prefixes with a status of
+        "container", get the number child prefixes. For all others, count child IP addresses.
+
+        Returns:
+
         """
         if self.status == Prefix.STATUS_CONTAINER:
             queryset = Prefix.objects.filter(prefix__net_contained=str(self.prefix), vrf=self.vrf)
             child_prefixes = netaddr.IPSet([p.prefix for p in queryset])
-            return int(float(child_prefixes.size) / self.prefix.size * 100)
+            return (child_prefixes.size, self.prefix.size)
```

Comment:
Anywhere you return a tuple, please return a `collections.namedtuple` instead.

See: https://gist.github.com/lucasnad27/e93b8f316259be3299e5#clarify-multiple-return-values-with-named-tuples